modules/claims_api/spec/fixtures/signature_b64.txt

```diff
@@ -0,0 +1 @@
iuLmlcXgBI7hdamk31vIO96WYD+vR8WrWqeqy+IOIa/cCKfBq8HxNLziFesaTNPHW0PxOlV0sg51fvtmjbZ3Lk/1PFBs65q9opU3N8kG15Y0V3opDkjsr5wBjp0bwvU71nag1OPU9nFNdaDHNp1Ogzzs794ke0tT55G2PvX2UT4JUzu5Ph9TaOfiDAJFo+XUnExe4/vezPXrRLZR/NlX69PvC+hd6L3wYXgHTATfqeoPgzj+TvJyHrwBfNUYHEodrPW6TqJucafFmh9Z0+E0Pa40z9xezRr9pqKPEUtxQJJ+fte4j+NLC+eo+9b50qRr0m+7rp2HoMw8OVhr7KqxfwTeCHuW1nZ204qvF0+nJ/9hsL11mciPGd/7SnTmx4HvXwPBfo+DwTUZngGdauoryvH9vvkWXAsfgn+B8WDAZjM5Gag6P5UdX4mzmtLAp7eNY1wOBtoccG5lB3BPCYzMZV0ttvGmSL/byV8EXwODwjm0i/21Q+xHtqf0267nIH1Uxme5ufro0jQ5myTGURNjH0neQXdRicRwGkp5Dfi/g6Iup9MTbUAoc2FKJ9dcZ4tK3sRgcHxfPUo+gP1YvA1mQ2R/MtuVwlWk+5R81lSKgyZZ8xxanFtaGcDO/7FS1vm2C46dfjbZHVz/ly0gBoG86CRX4KfYmb8w5sOHwcjaAuJ0HXI3tEWjadiIJ0fd34J9PwsL4Ivwe+AJTWA490NwOlwBlpeX9Dukd8EpsB88VcpLSH11LYX/Ap3S7ynwxtHRF8Pr4SOQPXljKk83yYDndDTz4A1wJnwDEhD1/lF3FdumfdcGKDNO0sHajYg+C85ijmfWreAJeBR0jsb7RzgWNgKd5I2iET3p94On24DQ+LX4qlD/fTAAToWz4bugk+LUdl+DxCBTnNN8ys65NRh8dTuKfUn6uHcD5GTwlfh1cP1i3SYwBWbA5nA9XApKxmhK3Z/uyXGyx+6tumvtq0/il+6thlHrwhVTiWNrR/nOPxpOgh9CxPbnwHxIv9SZdjOeDjXg7OumDQ6lm/FcgyQgDEbf8XvBIvCUZxyyQ5I6KO2o47cFb0nziuv7JdwHyyDSbV+pcz2uub0fX1nu3bHt7/ymHi5xXyvAANU+EcdTRjxAMnEzfbNYNxWn6XDzh4En60r4V9CAJ4A3ya1QBxLFlaJecYxsbrC2nYZreNSBUOfX0G3Qah3kOAm+wRr20842CQi/VWbCRNgMfgE6XB6HJyE2sd+GYDtva29IA3Ex/BSUdbHXZqQ+n07YS+oFeaW+Edyk0X4N3Au1QSj2lHq8ng17VGbNCbQeTfuuypimyWf8OHCwwdLHQ/RyOBi2gXvgTtDBGYtsX+LtMgEMkpvAW8V5hjoOXYZPXFBOfnsWg+LXWWq7HIIhPgh7djGI7SRBFJtqP0m5Hg915xbZn3RjC4jtxpy46Lwf3Ux7E2NuwcO4IB2kLRSddi4cZaGIrxJPu6nt2rRtl7Fo2hHHz6HztT2rUY/NwChre1EmOkJ0mOQkm8apbWdS1RF/bb2i5IeSbErjnVt9XUMtmfO1KHNr1PXDlm9H6rBNNEoDx9DdUt/X+bjOuztpr+VqM39l2Pb94Aelv5R2AZ3tR6TzObbiv8Z+e2Qu63eFqeAvEX+pfAVOgcHmN0D85TJi8kIPjNrh5lPWwPl1pTEHM7h13gqeRh2mY0XH+0vBn8gzwNeC4+0BT4A/K8fDRPg6eNWrWwKLwP9ArPNGsJ0/8xeCv0h8vXwLtL1BYUC5bsv+OjLveg0Gg8mf0amLDtXwSgw5vLOs3eiuLThC1pqT18vZmVEHeWJ1tA7fGnSGeVMdrywHT6Qn2F9d/pz8ecHT7q+CB0AHWW97PzYNlAsg4rhTwVeLfwT6q+1hGKoYrAaicigsANcwYuJGR1ri4DjdNE42zRVsPvpua9wYpadcdLzoEB1pMBgUcbZOiqMXk/f6fww8wd4AmZPsGqVe/2tK6yNId4RH4Ha4Dpxbsb127raXjGVdbgtvBYPCPm+D68GgGLHbgrlWnkLz60rcbMiYcXI346RNUg2gs+NgnexJ3wa2AOu8jp3Dk+vp9qTr7KVV+jPy/nPZjzhnnJT27bVa9iQbRDITDIIvwYVwN3ibRGxrn36Crt3WsfeGK8BgG9GgYL4BxlDXj8TxMaYGCL3661Ad7pWrwz3h/tvntW7Z97rO94TrcB3vqc5pX0beAMhJJ9tTXF/tdNeo1GuNrqkZ/OkJ9lQr74ZtYQ94J0yAn4DiHj3xBsRgY2dN1teBM4nyXmCAXQnWGTR5rZAdGXGza5IEge3cSE27r9e7jhWd7knX6ZuA73ONZn+vb0+zjn8I7gKNYQBYl2uYbE+JgbsFaNa5LoyaoJjGas6EBXBJWZnB/AB8HP4cDOhIbV/XqKMTNGmjjWbAVPCWuxo8AIr7Wxfr7ww2lEcMOpQ+Ot9TruM9/eY97RpBx+vUx0BHe7LdbMr9Xu2uS6Mo5nWyEmen3GiH95kTO4tpjoQLQMepz61wTtGTdALmEtLrwZuvLd6WO4OB8ErQXnfAzeDBUDJnUxqFZ6/AiEN0/nTYBvzZ5qJ1jCfba12n63yDoD4tFLtKTnldORoOr+cfLB8bTKDBcTC/NPRw6HQDI+LBMEBOg/HgwbgavgOLYSKMA3+l5Ja8k3xbMmdbP6LlXoGRhfgqEDe6AnpdbY6Xk0525QlP3vSFJDrb7wqd7eH4ANTiXr0xdwJvUNkN3goeIG+HyVDLtRRugfvgEfBQ+UrVrg/Cj0E7elhGTVzAUMU+cb6LzwaSDnW8sdw+DtLBl4GOvx3cv0Ejvh518BJYCvdA/QvFQ7UjbF/SbUm9cRzbW8eg8Pb14P0v/BgyL9nRERewJqnbvBid38/+s++ZNPY14SvT14HBoEO7SeyWvt3adNONelC4qCy+2wLX61ZZQDtJ/U2RWvW+NhSDoEZd+iZVp1hO0CTNx2ynwWg+/h8UiHR2cGRIyAAAAABJRU5ErkJggg==
```

Comment:
I would add something like `data:application/png;base64,...` to te start of this payload or something so that people know and understand what it is they are actually sending, it would follow the similar convention we've used for pdf on all of the file uploads portions. https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/spec/fixtures/base64pdf

Reply:
Lets chat about that. The only reason we did that in vba_docs was to be backward compatible and know when it was b64 vs binary. Which is why we have to strip it there before processing it. Since we dont have those requirements here, I am not sure that it is necessary.